publicsuffixlist.js

```diff
@@ -550,7 +550,18 @@ const fromSelfie = function(selfie, decoder) {
 const enableWASM = (( ) => {
     let wasmPromise;
 
-    const getWasmInstance = async function(wasmModuleFetcher, path) {
+    const wasmModuleFetcher = async () => {
+        const url = new URL('wasm/publicsuffixlist.wasm', import.meta.url);
```

Comment:
This uses [`import.meta.url`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta#using_import.meta).

On Node.js this should be a `file://` URL, which works with the fs API.

The objective of passing this as the second argument to the `URL` constructor is that we want the URL of the `.wasm` file relative to the URL of the module.